contracts/staking_contract.py

```diff
@@ -366,10 +370,144 @@ def add_admin(self, deployer: Account, proxy: ProxyNetworkProvider, address_to_w
+        if not raw_results:
+            return 0
+        return int(raw_results)
+    
+    def get_total_energy_for_week(self, proxy: ProxyNetworkProvider, week: int) -> int:
+        data_fetcher = StakingContractDataFetcher(Address(self.address), proxy.url)
+        raw_results = data_fetcher.get_data('getTotalEnergyForWeek', [week])
+        if not raw_results:
+            return 0
+        return int(raw_results)
+    
+    def get_total_rewards_for_week(self, proxy: ProxyNetworkProvider, week: int) -> int:
+        data_fetcher = StakingContractDataFetcher(Address(self.address), proxy.url)
+        raw_results = data_fetcher.get_data('getTotalRewardsForWeek', [week])
+        if not raw_results:
+            return 0
+        return int(raw_results)
+    
+    def get_remaining_boosted_rewards_to_distribute(self, proxy: ProxyNetworkProvider, week: int) -> int:
+        data_fetcher = StakingContractDataFetcher(Address(self.address), proxy.url)
+        raw_results = data_fetcher.get_data('getRemainingBoostedRewardsToDistribute', [week])
+        if not raw_results:
+            return 0
+        return int(raw_results)
+    
+    def get_undistributed_boosted_rewards(self, proxy: ProxyNetworkProvider, week: int) -> int:
+        data_fetcher = StakingContractDataFetcher(Address(self.address), proxy.url)
+        raw_results = data_fetcher.get_data('getUndistributedBoostedRewards', [week])
+        if not raw_results:
+            return 0
+        return int(raw_results)
+    
+    def get_farm_token_supply(self, proxy: ProxyNetworkProvider) -> int:
+        data_fetcher = StakingContractDataFetcher(Address(self.address), proxy.url)
+        raw_results = data_fetcher.get_data('getFarmTokenSupply')
+        if not raw_results:
+            return 0
+        return int(raw_results)
+    
+    def get_reward_capacity(self, proxy: ProxyNetworkProvider) -> int:
+        data_fetcher = StakingContractDataFetcher(Address(self.address), proxy.url)
+        raw_results = data_fetcher.get_data('getRewardCapacity')
+        if not raw_results:
+            return 0
+        return int(raw_results)
+    
+    def get_reward_reserve(self, proxy: ProxyNetworkProvider) -> int:
+        data_fetcher = StakingContractDataFetcher(Address(self.address), proxy.url)
+        raw_results = data_fetcher.get_data('getRewardReserve')
+        if not raw_results:
+            return 0
+        return int(raw_results)
+    
+    def get_accumulated_rewards(self, proxy: ProxyNetworkProvider) -> int:
+        data_fetcher = StakingContractDataFetcher(Address(self.address), proxy.url)
+        raw_results = data_fetcher.get_data('getAccumulatedRewards')
+        if not raw_results:
+            return 0
+        return int(raw_results)
 
     def contract_start(self, deployer: Account, proxy: ProxyNetworkProvider, args: list = []):
         _ = self.start_produce_rewards(deployer, proxy)
         _ = self.resume(deployer, proxy)
+        pass
```

Comment:
why you need this `pass`?

Reply:
In this case it does nothing, but it's absolutely needed to have when the other two lines are commented (when we're doing deploys that don't need start and resume automatically). Basically, you can't have empty code in function definitions and pass is satisfying this condition.
But yes, it's an ugly solution until we implement some feature with which we could specify whether we need contract start or not.